cypress/integration/pages/onDemandTV/index.js

```diff
@@ -16,7 +16,13 @@ Object.keys(config)
       describe(`${pageType} - ${currentPath}`, () => {
         before(() => {
           Cypress.env('currentPath', currentPath);
-          visitPage(currentPath, pageType);
+          let newPath;
+          if (Cypress.env('APP_ENV') === 'test') {
+            newPath = `${currentPath}?renderer_env=live`;
+          } else {
+            newPath = currentPath;
+          }
```

Comment:
As above you can use the dataEndpointOverride() function here instead

Reply:
```suggestion
          const newPath = dataEndpointOverride(path);
```

I am now thinking that the name of the function is not great... it could now be called something like overrideRendererOnTest or something?